scripts/environment/ci-setup-helm.sh

```diff
@@ -1,21 +1,13 @@
 #!/bin/bash
 set -euo pipefail
 
-if [[ -z "${CI:-}" ]]; then
-  echo "Aborted: this script is for use in CI, it may alter your system in an" \
-    "unwanted way" >&2
-  exit 1
-fi
-
-set -x
-
 KUBERNETES_VERSION="v1.18.6"
```

Comment:
Note to self: we need to figure out a way to manage this and keep it up to date, how that the files are moving around.